org.mixedrealitytoolkit.input/Subsystems/Hands/OpenXRHandsSubsystem.cs

```diff
@@ -25,7 +24,7 @@ namespace MixedReality.Toolkit.Input
         Name = "org.mixedrealitytoolkit.openxrhands",
         DisplayName = "Subsystem for OpenXR Hands API",
         Author = "Mixed Reality Toolkit Contributors",
-        ProviderType = typeof(OpenXRProvider),
+        ProviderType = typeof(HandsProvider<OpenXRHandContainer>),
```

Comment:
I may be missing something (bad eyes?) ... where is OpenXRHandContainer defined?

Reply:
They're existing private classes defined in their corresponding subsystem files:

https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/blob/62dca36d8fdae7c0c17910f10b1eca2f22ed3e70/org.mixedrealitytoolkit.input/Subsystems/Hands/OpenXRHandsSubsystem.cs#L54